Project.toml

```diff
@@ -1,6 +1,6 @@
 name = "SpecialFunctions"
 uuid = "276daf66-3868-5448-9aa4-cd146d93841b"
-version = "0.10.2"
+version = "0.10.3"
```

Comment:
Since this is exporting a new function, don't we need a bump of the minor version

Reply:
Nope

Reply:
got it, thanks for letting me know

Reply:
It would be if the major version is >=1, but since we're still at 0, minor bumps are only for breaking changes, which new exports aren't considered to be. Note that this is a bit different from how things were in the METADATA days, where folks were encouraged to make minor version bumps for any new features.